templates/partials/header.php

```diff
@@ -6,9 +6,26 @@
 
 /** @var string $html_title */
 /** @var string|callable $page_title */
+/** @var string $url */
+/** @var string $html_description */
 /** @var ?callable $sub_head */
 /** @var ?string[] $breadcrumbs */
+add_filter(
+	'gp_head',
+	function () use ( $html_title, $url, $html_description ) {
+		echo '<meta name="twitter:card" content="summary" />' . "\n";
+		echo '<meta name="twitter:site" content="@WordPress" />' . "\n";
+		echo '<meta name="twitter:title" content="' . esc_attr( $html_title ) . '" />' . "\n";
+		echo '<meta name="twitter:description" content="WordPress Translation Events" />' . "\n";
```

Comment:
Forgotten to update?
```suggestion
		echo '<meta name="twitter:description" content="' . esc_html( $html_description ) . '" />' . "\n";
```